README.md

```diff
@@ -15,22 +15,38 @@ Second, tool (1) requires that the git repository you wish to execute `byoqm` on
 Make sure that you have the following folder structure:
 
 ```
-<repo>
+<repository>
   .git
 experiment-toolkit
 ```
 
-Where `repo` is a local copy of repository that you wish to execute the experiment on.
+Where `repository` is a local copy of repository that you wish to execute the experiment on.
+
+Furthermore, you will need to have cloned the parameterIT/testing repository as a sibling directory as well, and have a seperate folder called 'target'. The final folder structure should look something like this: 
+```
+<repo>
+  .git
+experiment-toolkit
+testing
+target
+```
 
 To visualize the results you need to follow some set-up related to bokeh.
 The following [link](https://docs.bokeh.org/en/latest/docs/user_guide/output/export.html) explains the set-up on their documentation.
 
+This tool also requires the user to have poetry installed.
+
 ## Running
+First ensure that you are in the `experiment-toolkit` folder.
 
-First run `byoqm` on the local repository by executing
+Then run `byoqm` on the local repository by executing
 
 ```sh
-./run-byoqm-on-tags.sh
+./run-byoqm-on-tags.sh <repository-name>
+```
+e.g.
+```sh
+./run-byoqm-on-tags.sh flask
```

Comment:
Maybe also include a concrete example of the folder structure i.e. where <repo> is is flask